simulation_run.py

```diff
@@ -106,23 +106,13 @@ def run_scenario(current_scenario):
 
     for method in current_scenario.methods:
         print(method)
-        start = timer()
-        score_dict = contributivity_measures.compute_contributivity(method, 
-                                                                    current_scenario)
-        end = timer()
-        
-        for score_method in score_dict:
```

Comment:
We don't need to loop for the independent score anymore ? `Independant scores raw` and `Independant scores additive `?